packages/inventory/src/index.js

```diff
@@ -0,0 +1,7 @@
+import ConnectionLoader from "./loaders/connection"
+import InventoryService from "./services/inventory"
+import * as SchemaMigration from "./migrations/schema-migrations/1665748086258-inventory_setup"
+
+export const service = InventoryService
+export const migrations = [SchemaMigration]
+export const loaders = [ConnectionLoader]
```

Comment:
Suggestion(non-blocking): maybe for a different pr, I talked to @adrien2p and he suggested creating a defined shape for module exports.

Reply:
I'm thinking we could add it in a follow up pr 😄 no need to do anything about it here

Reply:
I agree with that, and we also will probably need to expand it a bit to integrate the module to the core as an option while loading it.